### PR TITLE
clone to new version for new flags for non-workspace based components

### PR DESCRIPTION
**Pull request by arolshan**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/applicationinsights/resource-manager/readme.md [hunk 1]
```diff
@@ -293,6 +293,7 @@ These settings apply only when `--tag=package-2018-05-01-preview` is specified o
 ``` yaml $(tag) == 'package-2018-05-01-preview'
 input-file:
 - Microsoft.Insights/preview/2018-05-01/componentProactiveDetection_API.json
+- Microsoft.Insights/preview/2018-06-17/components_API.json
```
Comment (MyronFanQiu) [inline specification/applicationinsights/resource-manager/readme.md]: should it be 2018-05-01?

File: specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2018-05-01/components_API.json [hunk 2]
```diff
@@ -0,0 +1,697 @@
...
+          "externalDocs": {
+            "url": "https://aka.ms/sl0yqd"
+          }
+        },
+        "ConnectionString": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Application Insights component connection string."
+        },
+        "RetentionInDays": {
```
Comment (MyronFanQiu) [inline specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2018-05-01/components_API.json]: will we support this one in this api version?

File: specification/applicationinsights/resource-manager/readme.md (hunk 1 above)
Comment (arolshan) [reply specification/applicationinsights/resource-manager/readme.md]: yes, sorry, good save

File: specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2018-05-01/components_API.json (hunk 2 above)
Comment (arolshan) [reply specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2018-05-01/components_API.json]: we support everything stable version supports

**Review by mmyyrroonn — APPROVED** [review]
Approved since it's just a copy

**Review by majastrz — APPROVED** [review]
Signed off from ARM side.

**Pull request merged**